### DISCUSSION TITLE
can i open a script with another script?

### DISCUSSION THREAD

**Discussion by cornernote** [johnlindquist/kit#1100] [Q&A] [answered]
Can we do something like this?

`script-1.js`

```js
let answer = await arg()
if (answer === 'run') {
  await exec(`open 'script-2.js'`)
}
```

`script-2.js`

```js
let anotherAnswer = await arg()
```

**Comment by cornernote** [comment]
`term()` kind of works, but you don't get the pretty interface of kit

**Comment by cornernote** [comment, marked as answer]
Got it :D 

```js
await exec('~/.kit/kar script-2')
```

**Reply by johnlindquist** [reply]
@cornernote try

await run("script-2")

**Reply by cornernote** [reply]
Thank you!  Both ways will come in handy.  Sometimes I want the output of `arg()` to go to `exec()`.  And sometimes I want shorter code.  `run()` is perfect!

Is run documented somewhere?  I was just reading docs/guides and stumbled over `~/.kit/kar`.

Also have to say, this is the coolest new software I've installed for a long while.  Well done it's awesome!
